### PR TITLE
Add note about notifications to DefaultStorIOSQLite.LowLevel javadoc

### LINKED ISSUE #702: Add note about notifications to javadoc of DefaultStorIOSQLite.LowLevel methods
Right now it is not mentioned that DefaultStorIOSQLite.LowLevel.insert, update, insertWithOnConflict, delete and executeSQL methods will not notify about changes in tables.
Especially `affectsTables` in `RawQuery` can be cause of misleading:

``` java
lowLevel
  .executeSQL()
  .withQuery(RawQuery.builder()
    .query("ALTER TABLE tweets ADD COLUMN number_of_retweets INTEGER")
    .affectsTables("tweets")
    .build());
```

### PR DESCRIPTION
**Pull request by nikitin-da**
Closes #702

### CODE REVIEW

File: storio-sqlite/src/main/java/com/pushtorefresh/storio/sqlite/impl/DefaultStorIOSQLite.java [hunk 1]
```diff
@@ -304,7 +304,13 @@ protected LowLevelImpl(@NonNull TypeMappingFinder typeMappingFinder) {
         }
 
         /**
-         * {@inheritDoc}
+         * Executes a single SQL statement that
+         * is NOT a SELECT/INSERT/UPDATE/DELETE on the database.
+         * <p>
+         * Notice: Direct call this method will not trigger notification from {@link RawQuery#affectsTables()}
```
Comment (artem-zinnatullin) [inline storio-sqlite/src/main/java/com/pushtorefresh/storio/sqlite/impl/DefaultStorIOSQLite.java]: of this method

File: storio-sqlite/src/main/java/com/pushtorefresh/storio/sqlite/impl/DefaultStorIOSQLite.java [hunk 2]
```diff
@@ -360,7 +373,16 @@ public Cursor query(@NonNull Query query) {
         }
 
         /**
-         * {@inheritDoc}
+         * Inserts a row into the database.
+         * <p>
+         * Notice: Direct call this method will not trigger notification
```
Comment (artem-zinnatullin) [inline storio-sqlite/src/main/java/com/pushtorefresh/storio/sqlite/impl/DefaultStorIOSQLite.java]: of this method

Comment (artem-zinnatullin) [inline storio-sqlite/src/main/java/com/pushtorefresh/storio/sqlite/impl/DefaultStorIOSQLite.java]: and comma after "notification"

File: storio-sqlite/src/main/java/com/pushtorefresh/storio/sqlite/impl/DefaultStorIOSQLite.java (hunk 1 above)
Comment (artem-zinnatullin) [inline storio-sqlite/src/main/java/com/pushtorefresh/storio/sqlite/impl/DefaultStorIOSQLite.java]: huh, yeah that's important piece of information!

File: storio-sqlite/src/main/java/com/pushtorefresh/storio/sqlite/impl/DefaultStorIOSQLite.java [hunk 3]
```diff
@@ -324,7 +330,14 @@ public void executeSQL(@NonNull RawQuery rawQuery) {
         }
 
         /**
-         * {@inheritDoc}
+         * Executes raw query on the database
+         * and returns {@link android.database.Cursor} over the result set.
+         * <p>
+         * Notice: Direct call this method will not trigger notification from {@link RawQuery#affectsTables()}.
```
Comment (artem-zinnatullin) [inline storio-sqlite/src/main/java/com/pushtorefresh/storio/sqlite/impl/DefaultStorIOSQLite.java]: of this method

File: storio-sqlite/src/main/java/com/pushtorefresh/storio/sqlite/impl/DefaultStorIOSQLite.java [hunk 4]
```diff
@@ -375,7 +397,25 @@ public long insert(@NonNull InsertQuery insertQuery, @NonNull ContentValues cont
         }
 
         /**
-         * {@inheritDoc}
+         * Inserts a row into the database.
+         * <p>
+         * Notice: Direct call this method will not trigger notification
```
Comment (artem-zinnatullin) [inline storio-sqlite/src/main/java/com/pushtorefresh/storio/sqlite/impl/DefaultStorIOSQLite.java]: of this method

File: storio-sqlite/src/main/java/com/pushtorefresh/storio/sqlite/impl/DefaultStorIOSQLite.java [hunk 5]
```diff
@@ -391,7 +431,16 @@ public long insertWithOnConflict(@NonNull InsertQuery insertQuery, @NonNull Cont
         }
 
         /**
-         * {@inheritDoc}
+         * Updates one or multiple rows in the database.
+         * <p>
+         * Notice: Direct call this method will not trigger notification
```
Comment (artem-zinnatullin) [inline storio-sqlite/src/main/java/com/pushtorefresh/storio/sqlite/impl/DefaultStorIOSQLite.java]: u got it :)

File: storio-sqlite/src/main/java/com/pushtorefresh/storio/sqlite/impl/DefaultStorIOSQLite.java [hunk 6]
```diff
@@ -407,7 +456,14 @@ public int update(@NonNull UpdateQuery updateQuery, @NonNull ContentValues conte
         }
 
         /**
-         * {@inheritDoc}
+         * Deletes one or multiple rows in the database.
+         * <p>
+         * Notice: Direct call this method will not trigger notification
```
Comment (artem-zinnatullin) [inline storio-sqlite/src/main/java/com/pushtorefresh/storio/sqlite/impl/DefaultStorIOSQLite.java]: while (true)

File: storio-sqlite/src/main/java/com/pushtorefresh/storio/sqlite/impl/DefaultStorIOSQLite.java (hunk 5 above)
Comment (nikitin-da) [inline storio-sqlite/src/main/java/com/pushtorefresh/storio/sqlite/impl/DefaultStorIOSQLite.java]: =)

**Pull request merged**